include/BALL/VIEW/RENDERING/RENDERERS/glRenderer.h

```diff
@@ -38,13 +38,13 @@
 #endif
 
 #ifdef BALL_OS_DARWIN
+        #include <OpenGL/gl.h>
 	#include <OpenGL/glu.h>
 #else
+        #include <GL/gl.h>
 	#include <GL/glu.h>
 #endif
 
-#include <QtGui/QOpenGLFunctions_1_3>
```

Comment:
Do you think this is a show-stopper for the moment or do you think we should proceed and merge changes into the master? I have the feeling that we do not want to further add fixes to a master branch that is based on Qt4.